python/mdvtools/stress_tests/test_json_write.py

```
@@ -0,0 +1,263 @@
+                        }
+                    }
+                }
+            })
+            
+        return data
+    
+    def validate_json_file(self, path):
+        """Check if a JSON file is valid and complete"""
+        try:
+            with open(path, 'r') as f:
+                data = json.load(f)
```

Ruff (0.8.2)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)